### ISSUE TITLE
tests doesn't compile

### ISSUE THREAD

**Issue by lifeishard** [OpenVPN/ovpn-dco#58]
make tests
make -C tests
make[1]: Entering directory `/home/ec2-user/ovpn-dco/tests'
cc -Wall ovpn-cli.c -I../include/uapi \
        `pkg-config --cflags --libs libnl-3.0 libnl-genl-3.0` \
        -lmbedtls -lmbedcrypto -Wall -o ovpn-cli
ovpn-cli.c: In function ‘ovpn_handle_peer’:
ovpn-cli.c:745:17: error: ‘OVPN_GET_PEER_RESP_ATTR_RX_BYTES’ undeclared (first use in this function); did you mean ‘OVPN_GET_PEER_RESP_ATTR_VPN_RX_BYTES’?
  if (attrs_peer[OVPN_GET_PEER_RESP_ATTR_RX_BYTES])
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                 OVPN_GET_PEER_RESP_ATTR_VPN_RX_BYTES
ovpn-cli.c:745:17: note: each undeclared identifier is reported only once for each function it appears in
ovpn-cli.c:749:17: error: ‘OVPN_GET_PEER_RESP_ATTR_TX_BYTES’ undeclared (first use in this function); did you mean ‘OVPN_GET_PEER_RESP_ATTR_RX_BYTES’?
  if (attrs_peer[OVPN_GET_PEER_RESP_ATTR_TX_BYTES])
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                 OVPN_GET_PEER_RESP_ATTR_RX_BYTES
ovpn-cli.c:753:17: error: ‘OVPN_GET_PEER_RESP_ATTR_RX_PACKETS’ undeclared (first use in this function); did you mean ‘OVPN_GET_PEER_RESP_ATTR_VPN_RX_PACKETS’?
  if (attrs_peer[OVPN_GET_PEER_RESP_ATTR_RX_PACKETS])
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                 OVPN_GET_PEER_RESP_ATTR_VPN_RX_PACKETS
ovpn-cli.c:757:17: error: ‘OVPN_GET_PEER_RESP_ATTR_TX_PACKETS’ undeclared (first use in this function); did you mean ‘OVPN_GET_PEER_RESP_ATTR_RX_PACKETS’?
  if (attrs_peer[OVPN_GET_PEER_RESP_ATTR_TX_PACKETS])
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                 OVPN_GET_PEER_RESP_ATTR_RX_PACKETS
ovpn-cli.c: In function ‘ovpn_send_data’:
ovpn-cli.c:874:27: error: ‘OVPN_CMD_PACKET’ undeclared (first use in this function); did you mean ‘OVPN_CMD_DEL_KEY’?
  ctx = nl_ctx_alloc(ovpn, OVPN_CMD_PACKET);
                           ^~~~~~~~~~~~~~~
                           OVPN_CMD_DEL_KEY
ovpn-cli.c:878:37: error: ‘OVPN_ATTR_PACKET’ undeclared (first use in this function); did you mean ‘OVPN_CMD_PACKET’?
  attr = nla_nest_start(ctx->nl_msg, OVPN_ATTR_PACKET);
                                     ^
ovpn-cli.c:37:24: note: in definition of macro ‘nla_nest_start’
  nla_nest_start(_msg, (_type) | NLA_F_NESTED)
                        ^~~~~
make[1]: *** [ovpn-cli] Error 1
make[1]: Leaving directory `/home/ec2-user/ovpn-dco/tests'
make: *** [tests] Error 2

**Comment by ordex** [comment]
Hi and thanks for the report.
This tool was not used that much anymore and therefore some changes were not committed yet.
I have fixed those errors now. You can pull master.

**Comment by lifeishard** [comment]
Thanks for fixing that bug, tests compiled now. It is my first time compiling kernel module so I wanted to make sure it works. Unfortunately when I try to run netns-test.sh I get the following error

netlink reports error (-12): Object not found
failed to send netlink message: No such file or directory (-2)
cannot set key

Should i start a new issue?

**Comment by ordex** [comment]
@lifeishard honestly dco and the test cli are going through a major restructuring, therefore fixing these old test scripts is not a priority right now as they will all be changed soon.
